assembly/pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.tachyonproject</groupId>
     <artifactId>tachyon-parent</artifactId>
-    <version>0.9.0-RC1</version>
+    <version>0.9.0-RC2</version>
```

Comment:
i guess without specifying the parent version, it can not inherit a variable from parent pom.